package.json

```diff
@@ -196,9 +205,27 @@
       "jest-localstorage-mock"
     ]
   },
+  "lint-staged": {
+    "linters": {
+      "*.{js,jsx,mjs}": [
+        "npm run lint:fix",
```

Comment:
I'm not sure with this. I think I'd change to `lint` with-out fix to avoid introducing a problem during committing. 

What do you think? Or should we use `lint:fix` and `lint` afterwards to check that the fix doesn't introduce an issue.

Reply:
Yeah it is wrong. We should do lint without fixing.